README.md

```diff
@@ -66,9 +66,6 @@ Optionally configure one of the following:
 
 **HS_THEME**: Either `dark` or `funky`. Default is `dark`.
 
-**HS_FULLSCREEN**: Run (1) or don't run (0) in fullscreen. Default is
-  no fullscreen.
-
```

Comment:
When did we decide to remove this option? I recall we agreed to leave it.

Reply:
Maybe I misunderstood the outcome of our conversation. Do you want me to add it back?

Reply:
No, but let's hold a moment of silence for yet another feature removed from this software.